### DISCUSSION TITLE
New version prompt that cannot be turned off

### DISCUSSION THREAD

**Discussion by StepaniaH** [GothenburgBitFactory/taskwarrior#3735] [Community support] [answered]
I am a beginner who just downloaded and started using taskwarrior, so this question may be very silly。
After downloading and configuring according to the documentation, I created two new tasks, and when I executed `task next`, the returned content included a prompt saying, `Recently upgraded to 3.3.0. Please run 'task news' to read highlights about the new release.` 
Then I used the `task news` command to check the version information, but no matter what, this prompt continues to exist. 
I tried to confirm through `task show` that the only changed values are version, the DATA path, and hooks path, and they all in the correct location. 
Please help me, how can I turn off this new version prompt!
```diagnostics
task 3.3.0
   Platform: Darwin

Compiler
    Version: Apple LLVM 16.0.0 (clang-1600.0.26.4)
       Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
 Compliance: C++17

Build Features
      CMake: 3.31.2
    libuuid: libuuid + uuid_unparse_lower
 Build type: Release

Configuration
       File: /Users/<name>/.config/task/taskrc (found), 860 bytes, mode 100644
       Data: /Users/<name>/.local/share/task (found), dir, mode 40755
         GC: Enabled
    $EDITOR: nvim
Hooks
     System: Enabled
   Location: /Users/<name>/.config/task/hooks
             (-none-)

Tests
   Terminal: 162x41
 Broken ref: Scanned 2 tasks for broken references:
             No broken references found
```

**Comment by djmitche** [comment, marked as answer]
I believe https://github.com/GothenburgBitFactory/taskwarrior/pull/3734 fixes this.

Was this built from homebrew? If not, and if you can try building from the latest `develop` and verify it's fixed, that would be great!

You can also hide this warning with the `news` item in the `verbose` configuration.

**Reply by StepaniaH** [reply]
Fine, I read this issue, and change the version in config file from 3.3.0 to 3.4.0, to make sure it satisfy the `news_version >= current_version`, and now this prompt is turned off.
